pyChilizer.tab/Project.panel/colorizers2.stack/Reset Filters.pulldown/Reset Overrides.pushbutton/script.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+from pyrevit import revit, DB
+from pyrevit import forms
+from pychilizer import database
+
+view = revit.active_view
+doc = revit.doc
+
+view_filter_ids = view.GetFilters()
+
+
+
+def overrides_reset():
+
+    # returns Override Graphic Settings object with all overrides set to initial values
+    no_color = DB.Color.InvalidColorValue
+    invalid_id = DB.ElementId.InvalidElementId
+
+    overrides = DB.OverrideGraphicSettings()
```

Comment:
you can simplify: 


```python
# erase overrides
override = DB.OverrideGraphicSettings().Dispose()
# get clear graphics without overrides
override= DB.OverrideGraphicSettings()
```

and maybe this only is enough

`override= DB.OverrideGraphicSettings()`

Reply:
fab, thanks for the tip JM! 🙌

Reply:
Nice one! Thanks @jmcouffin  Closing now